README.md

```diff
@@ -131,7 +96,9 @@ For History service, just set all needed environment variables and execute:
 
 Persister is much simpler, just execute:
 ```bash
-python -m history.subscriber.kafkaSubscriber
+$ python -m history.subscriber.kafkaSubscriber
+#if the above command doesn't work, try:
```

Comment:
Why it could not work? :sweat_smile:
It's the user instructions, must be assertive :dart: